packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -1,6 +1,6 @@
 import { setTimeout as sleep } from 'node:timers/promises';
 import { AsyncQueue } from '@sapphire/async-queue';
-import fetch, { RequestInit, Response } from 'node-fetch';
+import { fetch, RequestInit, Response } from 'undici';
```

Comment:
```suggestion
import { fetch, type RequestInit, type Response } from 'undici';
```

Reply:
fixed in c8fdd310579c6473fd5f0423c04dd636ec2d5fce